src/Parsers/ASTDropQuery.h

```diff
@@ -40,6 +40,9 @@ class ASTDropQuery : public ASTQueryWithTableAndOutput, public ASTQueryWithOnClu
     // We detach the object permanently, so it will not be reattached back during server restart.
     bool permanently{false};
 
+    /// Example: Drop TABLE t1, t2, t3...
```

Comment:
To the uninitiated reader, `Example` could mean that `TRUNCATE t1, t2, ...` will also work, but as far as I understand, `database_and_tables` is used *only* for `DROP TABLE`. Please make this more clear.